unlock-app/src/middlewares/postOfficeMiddleware.ts

```diff
@@ -1,4 +1,4 @@
-import { Action } from 'redux'
+import { Action } from '../unlockTypes'
```

Comment:
Is the use of `Action` here vs redux intentional?

Reply:
Yep, that one was an accidental auto-import. Ours is a superset of the redux-provided type, which allows us to access arbitrary keys on the action.